### DISCUSSION TITLE
Snippets - Templates

### DISCUSSION THREAD

**Discussion by karlb690** [Zettlr/Zettlr#3335] [Q&A]
Hi,

I'm using snippets to add a YAML block to the beginning of new documents I create.

My question is - is there a way to create a new document that is already populated with the YAML block (or other text)? That is, create a pre-populated document (like you would from a template)?

Thank you.

**Comment by Etienne-TR** [comment]
I'm very interested in this functionality too !

It would be useful for instance to create a new note with default created-time and last-edited-time yaml properties.

**Comment by Etienne-TR** [comment]
Or to use snippet as a template for meetings note, daily note, etc.

A shortcut as CTRL+SHIFT+N could be used to open a selection list of snippets (that could be filtered by taping few characters as for the list of notes after having taped `[[`).
